ledger/catchupaccessor.go

```diff
@@ -356,15 +395,19 @@ func (c *catchpointCatchupAccessorImpl) processStagingContent(ctx context.Contex
 	wdb := c.ledger.trackerDB().Wdb
 	start := time.Now()
 	ledgerProcessstagingcontentCount.Inc(nil)
+
 	err = wdb.Atomic(func(ctx context.Context, tx *sql.Tx) (err error) {
 		crw := store.NewCatchpointSQLReaderWriter(tx)
 		arw := store.NewAccountsSQLReaderWriter(tx)
-
+		err = crw.WriteCatchpointStateUint64(ctx, store.CatchpointStateCatchupVersion, fileHeader.Version)
+		if err != nil {
+			return fmt.Errorf("CatchpointCatchupAccessorImpl::processStagingContent: unable to write catchpoint catchup state '%s': %v", store.CatchpointStateCatchupVersion, err)
```

Comment:
nits: `unable to write catchpoint catchup state` -> `unable to write catchpoint catchup version `